### ISSUE TITLE
I don't see a pointer/false guess and the google maps opener does not work.

### ISSUE THREAD

**Issue by lennsteenbergen** [dharmik2319/geotastic-helper#3]
I don't see a pointer/false guess when using violentmonkey on Firefox (Fedora Workstation, Linux). The function with the coordinates works good, but the map opener (google maps) does not work. Can you fix this for me? I hope it is not too hard/difficult.

~Lenn
![Screenshot from 2024-05-08 16-52-04](https://github.com/dharmik2319/geotastic-helper/assets/125652184/ee7674c0-8d89-4c31-adad-56c321f0ce9b)

**Comment by dharmik2319** [comment]
also, look around in the console logs to see if you get any errors from violentmonkey. if you have the ability, try debugging by putting console.logs at places in the script

**Comment by dharmik2319** [comment]
Any updates @Lengogame ?

**Comment by JustMercyV** [comment]
Hey, I'm experiencing the same issue. I use chrome, installed the script using Violentmonkey & using the install with URL function.
I already tried reloading the website, restarting my browser, I even restarted my PC but nothing seems to work. I'm not experienced with stuff like this at all, but some support would be great!
@dharmik2319

**Comment by dharmik2319** [comment]
what url did you use to install it?

**Comment by dharmik2319** [comment]
try installing it on firefox and report back.

**Comment by JustMercyV** [comment]
> what url did you use to install it @JustMercyV? and do you see any relevant errors in the console? and what mode are you playing?

I installed the script using the https://github.com/dharmik2319/geotastic-helper/raw/master/geotastic-helper.user.js url, and I'm trying to play the casual Worldwide Pinpointing mode from Quickplay. I tried this on Firefox as well, but nothing changed. 
I don't really know what you mean with relevant errors in the console sadly, so I guess I won't be able to answer that. I can still use the 1 and 2 functions (coordinates & open google maps in a new window), but it doesn't show the fake pointer.

**Comment by dharmik2319** [comment]
Hmm, all these issue seem weird to me. The extension works just fine for me (on Firefox and Brave (chromium based)) and also on other devices.

**Comment by ZeonTro2** [comment]
Yeah it doesnt seem to work for me too has it been fixed and how much progress has been made? @dharmik2319

**Comment by dharmik2319** [comment]
no, it is not fixed yet. and i should probably add a disclaimer in the readme now.

**Comment by ZeonTro2** [comment]
@dharmik2319 oh alright, goodluck trying to fix it

**Comment by Nick-78** [comment]
hey, today i saw the poiter false, but now i cant see it anymore, what should i do?

**Comment by Nick-78** [comment]
@dharmik2319 ?

**Comment by dharmik2319** [comment]
i am busy these days. and Geotastic changes too fast for me to update it. for now be satisfied with jist the coords (press 1)

**Comment by Nick-78** [comment]
ok, thank you for ur work, but u know why it has appeared to me yesterday?

**Comment by dharmik2319** [comment]
yes, i am aware of it happening. it might be because the mode was different. or you were lucky with the timing (check code to understand what i mean)

**Comment by Nick-78** [comment]
ok thank you so much

**Comment by dharmik2319** [comment]
@Spad0k  just press 1, it should work.

**Comment by jackthebackpack** [comment]
any news on the marker. i dont know why but randomly worked for me then i reinstalled it now it isnt working.

**Comment by dharmik2319** [comment]
lol thats the thing. the site gets updated too frequently, and it is a pain to read the obfuscated, minified code and reimplement things. and the most essential part of the program, hijacking the google module, is a race condition. and anyways, these days i am really busy. ig you will need to wait till next year. sorry for the delays. you can submit a pr if you find something out, tho.

**Comment by lennsteenbergen** [comment]
Sorry for the (very) late reply. Will be active from now on. I think this issue will be fixed by #9. Take your time fixing the code!
